src/constants/translates.js

```diff
@@ -19,3 +19,37 @@ export const DOC_STATUSES_TRANSLATES = createGetters({
   [DOC_STATUSES.NOT_VERIFIED]: $t('constants.documentStatus.notVerified'),
   [DOC_STATUSES.VERIFIED]: $t('constants.documentStatus.verified'),
 });
+
+export const SCOPES_TRANSLATES = createGetters({
+  // Offline access
+  offlineAccess: $t('components.scopes.offlineAccess'),
+  // User
+  user: $t('components.scopes.user.title'),
+  userEmailRead: $t('components.scopes.user.email'),
```

Comment:
why we can't use keys from `SCOPES` object? isn't it exists in `auth`?

Reply:
better to use `SCOPES` constants from vault.
and not make them as camelCase, stay them in their values, like `user:email:read`.
because it's hard to link with `userEmailRead` and understand that second is scope name but compiled somewhere in code